industrial_taxonomy/pipeline/official/gdp/utils.py

```diff
@@ -0,0 +1,21 @@
+"""Clean and process GDP data"""
+
+import pandas as pd
+import re
+
+YEAR_RANGE = range(1998, 2020)
+
+
+def process_gdp_table(table: pd.DataFrame):
+    """Cleans up the gdp table"""
```

Comment:
The docstring doesn't tell us anything more than the function name
```suggestion
    """Drop rows without LA codes and tidy column names."""
```